### DISCUSSION TITLE
Scrypted

### DISCUSSION THREAD

**Discussion by ksj9877us** [community-scripts/ProxmoxVE#716] [Request script]
### Application Name

[Script Request] Scrypted

### Website

https://docs.scrypted.app/

### Description

[Scrypted](https://scrypted.app/) is an [open source](https://github.com/koush/scrypted) video integration platform. Using various [Plugins](https://docs.scrypted.app/platforms.html), Scrypted can receive streams from a range of cameras and send them to other platforms ([HomeKit](https://docs.scrypted.app/homekit.html), [Google Home](https://docs.scrypted.app/google-home.html), [Alexa](https://docs.scrypted.app/alexa.html), [Home Assistant](https://docs.scrypted.app/home-assistant.html)).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
2024-11-25
Changed
💥 Breaking Changes
Remove Scrypted script @MickLesk (#511)
Because of request from Scrypted maintainer: #494
Official Scrypted script can be used instead: https://docs.scrypted.app/installation.html#proxmox-ve

**Comment by g0tmk** [comment]
what is the opinion of the maintainers to add a scrypted entry that simply links to the scrypted docs? proxmox LXC is the primary install method so they provide a script

Personally I just like the idea of the helper-scripts site being a one stop shop for everything I will want to install on my host, so I think an entry would be useful, even if the script itself is not maintained here

this is the link I would recommend using: 
https://docs.scrypted.app/installation.html#proxmox-ve

**Reply by MickLesk** [reply]
No, that will not happen. We can't guarantee that external projects will be vulnerable or faulty, which would cause even more problems for us.

In addition, the instructions and the script from them do not work properly at all.

**Reply by g0tmk** [reply]
Fair enough, thanks for the reply.
